### DISCUSSION TITLE
Error in Quiz: 5.5 Organizing Your Data Loaders with Data Modules (Part 2)

### DISCUSSION THREAD

**Discussion by ghost** [Lightning-AI/dl-fundamentals#86] [Lecture videos and quizzes]
All checkboxes can be selected even the one that is valid base on the question. Need to change the answers select only invalids

**Comment by rasbt** [comment]
Ah yes, thanks for bringing this up. I think this is the same issue as #17 . Our web developer is going to take a look at this hopefully some time
